docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -466,3 +466,107 @@ function hs_paragraph_types_update_10013(&$sandbox) {
 
   $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
 }
+
+/**
+ * Imports config for field_spotlight_style.
+ */
+function hs_paragraph_types_update_10014() {
+
+  // Load the configuration from the config/sync directory.
+  $config_directory = new FileStorage(Settings::get('config_sync_directory'));
+
+  $entity_type_manager = \Drupal::entityTypeManager();
+
+  // Update field_spotlight_style field config.
+  $field_config = $entity_type_manager->getStorage('field_config');
+  $config_data = $config_directory->read('field.field.paragraph.hs_spotlight.field_spotlight_style');
+  $existing_config = $field_config->load($config_data['id']);
+  if ($existing_config) {
+    $existing_config = $field_config
+      ->updateFromStorageRecord($existing_config, $config_data);
+    $existing_config->save();
+  }
+
+  // Update field_spotlight_style field storage.
+  $field_storage_config = $entity_type_manager->getStorage('field_storage_config');
+  $config_data = $config_directory->read('field.storage.paragraph.field_spotlight_style');
+  $existing_config = $field_storage_config->load($config_data['id']);
+  if ($existing_config) {
+    $existing_config = $field_storage_config
+      ->updateFromStorageRecord($existing_config, $config_data);
+    $existing_config->save();
+  }
+}
+
+/**
+ * Updates spotlight paragraphs with new default values.
```

Comment:
```suggestion
 * Update spotlight paragraphs with new default values.
```

This is pedantic, but generally update descriptions are not written using this tense.